src/sec_certs/dataset/cc.py

```diff
@@ -271,7 +274,13 @@ def _set_local_paths(self):
                 self.targets_txt_dir,
                 self.certificates_txt_dir,
             )
-        # TODO: This forgets to set local paths for other auxiliary datasets
+
+    def process_auxiliary_datasets(self, download_fresh: bool = False) -> None:
+        self.aux_handlers[CCMaintenanceUpdateDatasetHandler].certs_with_updates = [  # type: ignore
+            x for x in self if x.maintenance_updates
+        ]
+        self.aux_handlers[CCSchemeDatasetHandler].only_schemes = {x.scheme for x in self}  # type: ignore
```

Comment:
It seems a bit weird that these two handlers have special stuff here.

Reply:
Not sure how to take care of this differently. The Handlers cannot access the `Dataset` instance they're bound to. And if they must be provided with some specific configuration (e.g., to avoid downloading data for all schemes, when only some are in the dataset), this information must be injected into the handlers.